README.md

```diff
@@ -15,6 +15,24 @@ This integrates the two with each other. Just click on a \*.kdbx file in Your Ne
 
 To update to a new version, simply repeat these steps.
 
+## Mimetype detection
+
+Unfortunately, apps can't declare new mimetypes on the fly. To make
+Keeweb work properly, you need to add a new mimetype in the
+`mimetypemapping.json` file (at Nextcloud level).
+
+To proceed, just copy `/resources/config/mimetypemapping.dist.json` to
+`/config/mimetypemapping.json` (in the `config/` folder at Nextcloud’s
```

Comment:
Are you sure you need to actually copy this file? The official docs seem to suggest it's not necessary.

Reply:
Well, of course you don't need to copy this file, but the file must be named `/config/mimetypemapping.json` in the end and it is a good idea to use the distributed file as a template so you can see which MIME types are already defined and what syntax to use. Also see [here](https://docs.nextcloud.com/server/14/admin_manual/configuration_mimetypes/index.html):

> By default Nextcloud is distributed with nextcloud/resources/config/mimetypealiases.dist.json. Do not modify this file, as it will be replaced when Nextcloud is updated. Instead, create your own nextcloud/config/mimetypealiases.json file with your custom aliases. Use the same syntax as in nextcloud/resources/config/mimetypealiases.dist.json.

If you have a suggestions for a better explanation how to add a custom MIME type, I will use that as tell.